.idea/gradle.xml

```diff
@@ -15,7 +15,6 @@
           </set>
         </option>
         <option name="resolveModulePerSourceSet" value="false" />
-        <option name="useQualifiedModuleNames" value="true" />
```

Comment:
is this change committed by mistake ?

Reply:
Oh no I did not make any changes in this file I do not know why is it showing

Reply:
okay np, just revert this change and push.

Reply:
How do I undo this? Should I delete the last commit and make another one by adding this 18th line?

Reply:
either you can manually add that line back again
or do it from studio
![image](https://user-images.githubusercontent.com/20037625/123247011-0b3c0380-d504-11eb-90d6-27c84e00484a.png)
git > choose the commit > choose file > revert change > commit > push